README.md

```diff
@@ -217,6 +219,7 @@ public class Calculator {
 
 Now that we've looked at how to create commits in GitHub for better version control, let's look at the best practices.
 
+*__\[This whole section is too similar to the source material\]__*
```

Comment:
This whole sections is valuable, but it is too close to the source (Git Tower) material. 

Following on my suggestions above: I would recommend that you read the recommendations in the Git Tower article carefully, then consider how you can modify your examples above to meet these expectations. Then you can explain each commit individually, with reference / citations back to the Git Tower article where appropriate. 

For example, if in one of the commit examples you mention that commits should be _complete_ then you can reference the Git Tower article in your explanation.